examples/nats-basic-client/publisher.bal

```diff
@@ -12,7 +12,7 @@ public function main() {
     // Initializes a producer.
     nats:Connection connection = new("nats://localhost:4222");
     nats:Producer producer = new(connection);
-    while (message != ESCAPE) {
+    while (true) {
```

Comment:
why are we change the condition to true here

Reply:
As this condition is checked again inside the `if` condition, we decided to remove this.

Reply:
shall we remove the if condition?. yes, It doesn't make sense to have if condition.

Reply:
Resolved